integration/sandbox_run_rollback_test.go

```diff
@@ -318,6 +318,74 @@ func sbserverSandboxInfo(id string) (*criapiv1.PodSandboxStatus, *types.SandboxI
 	}
 	return status, &info, nil
 }
+// TestRunPodSandboxNoCNIPlugins checks if the sandbox creation fails with the expected error message when no CNI plugins are initialized.
+func TestRunPodSandboxNoCNIPlugins(t *testing.T) {
+	if runtime.GOOS != "linux" {
+		t.Skip()
+	}
+
+	testCase := func(restart bool) func(*testing.T) {
+		return func(t *testing.T) {
+			t.Log("Init PodSandboxConfig with specific label")
+			labels := map[string]string{
+				t.Name(): "true",
+			}
+			sbConfig := PodSandboxConfig(t.Name(), "nocni", WithPodLabels(labels))
+
+			t.Log("Inject CNI plugin initialization failpoint")
+			injectCNIFailpoint(t, sbConfig, map[string]string{
```

Comment:
can make the following change to resolve the syntax error:
```			
conf := &failpointConf{
		Add: "1*error(no CNI plugins initialized))",
}
injectCNIFailpoint(t, sbConfig, conf)
```

Reply:
could you also please run `gofmt -w -s <containerd-path>/integration/sandbox_run_rollback_test.go` after making the necessary changes to resolve the linter error?